docs/modeling/model-components/attributes/simple-model.md

```diff
@@ -0,0 +1,37 @@
+# [SimpleModel]
+
+`IntelliTect.Coalesce.SimpleModelAttribute`
+
+Used to mark a class as a [Simple Model](/modeling/model-types/simple-models.md) for generation by Coalesce.
+
+## Usage
+
+The `[SimpleModel]` attribute is used to explicitly include a standalone class as a Simple Model in your Coalesce application. This is particularly useful for data transfer objects, configuration objects, or other data classes that are not naturally discovered through the typical Coalesce discovery process.
+
+``` c#
+[SimpleModel]
```

Comment:
This new attribute must be used alongside [Coalesce], just like [Service] which I told you to model this off of.